src/main/kotlin/mb/coronium/plugin/FeaturePlugin.kt

```diff
@@ -84,14 +82,6 @@ class FeaturePlugin : Plugin<Project> {
         }
         builder.version = project.eclipseVersion
       }
-      // Add dependencies from Gradle project.
-      for(dependency in bundleConfig.dependencies) {
-        if(dependency.version == null) {
-          error("Cannot convert dependency $dependency to a feature dependency, as it it has no version")
-        }
-        val version = MavenVersion.parse(dependency.version!!).toEclipse()
-        builder.dependencies.add(Feature.Dependency(Feature.Dependency.Coordinates(dependency.name, version), false))
-      }
```

Comment:
Here we would add Gradle dependencies to the feature. There's two problems:

1. We don't check if the dependency already exists in the feature.xml.
2. This happens during configuration, which means the `dependency.version` may be unspecified.